packages/build/src/plugins_core/main.js

```diff
@@ -1,25 +1,63 @@
+const { isDirectory } = require('path-type')
+
+const { LOCAL_INSTALL_PLUGIN_NAME } = require('../install/local')
+
 const FUNCTIONS_INSTALL_PLUGIN = `${__dirname}/functions_install/plugin.js`
 const FUNCTIONS_PLUGIN = `${__dirname}/functions/plugin.js`
 
-const { LOCAL_INSTALL_NAME } = require('../install/local')
+// List of core plugin names
+const FUNCTIONS_PLUGIN_NAME = '@netlify/plugin-functions-core'
+const FUNCTIONS_INSTALL_PLUGIN_NAME = '@netlify/plugin-functions-install-core'
+const EDGE_HANDLERS_PLUGIN_NAME = '@netlify/edge-functions-deploy-plugin'
```

Comment:
the name should be `@netlify/edge-handlers-deploy-plugin`. We still need to change the package generation, but it should happen soon.